src/Sylius/Component/Attribute/AttributeType/SelectAttributeType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * Class SelectAttributeType
+ * @package Sylius\Component\AttributeBundle\AttributeType
```

Comment:
It seems You've missed two author blocks, here and in the [spec](https://github.com/Sylius/Sylius/pull/4397/files#diff-d99a44eeca3bfed469ab9cdf0dad4df3R18).